app/api/server/v1/rooms.js

```diff
@@ -407,6 +407,10 @@ API.v1.addRoute('rooms.export', { authRequired: true }, {
 				throw new Meteor.Error('error-invalid-format');
 			}
 
+			if (dateFrom > dateTo) {
+				throw new Meteor.Error('error-invalid-dates');
```

Comment:
Can you please add a translation (or descriptive message) to this error? :D

Reply:
@KevLehman I have added a proper descriptive message for the error. Please review 😁

Reply:
When I talked about translation, i was referring to add an entry on the `i18n` files for the error message (at least in english) 😬 

https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-i18n/i18n/en.i18n.json

Reply:
Oh sorry, I didn't get you @KevLehman. I have made the changes accordingly.  Hopefully this time it is right. Please Review 😀